examples/reactive-serial-circ.rs

```diff
@@ -0,0 +1,76 @@
+//! Blinks an LED
```

Comment:
I don't see any blinking led in this UART DMA demo ;-)

Reply:
![green_bl](https://user-images.githubusercontent.com/3072754/49588836-d6352900-f978-11e8-811d-1a72c794ff65.gif)